ruby_fundamentals/ruby_basics.md

```diff
@@ -0,0 +1,127 @@
+## Ruby Basics
+
+Let's try to run some Ruby code!
+
+
+- Open a terminal
+
+```
+irb
+```
+
+##### Basic example on how to print text
+```
+puts 'Hello World!'
+p 'Hi! I am here.'
+```
```

Comment:
Can insert extra challenge/question here e.g. "What is the difference between `puts` and `print`"?